functions/api/action.js

```diff
@@ -237,11 +297,23 @@ function validateParams(body, type) {
                 toBranchName: joi.string().min(1).max(30).required(),
                 product: joi.string().alphanum().length(20).required(),
                 productName: joi.string().min(1).max(30).required(),
-                operationalQuantity: joi.number().required(),
+                operationalQuantity: joi.number().integer().strict().required(),
                 note: joi.string(),
                 pendingRequestsId: joi.string().alphanum().length(20).required(),
             })
             break
+        case constants.MOVE:
```

Comment:
Are there 2 different operations transferProduct and moveProduct?

Reply:
Not sure on what name to track transfer from headoffice to branch(doesnt need pendingRequests). So named it "move"

Reply:
if any better name, ill change it.